docs/editors-guide/f-entities.md

```diff
@@ -193,6 +193,7 @@ MONDO:entailed  |   An inferred superclass (which is a redundant axiom) |   subc
 MONDO:equivalentObsolete    |   Used for cases where we have exact 1:1 matches between a live class in Mondo and an obsolete class in the source ontology. We want to avoid making an equivalence axiom (MONDO:equivalentTo) here.   |   xrefs   |   N   |   MONDO:0020499 'Nipah virus disease' database_cross_reference: Orphanet:1239 (was obsoleted in Orphanet) {source=MONDO:equivalentObsolete}
 MONDO:equivalentTo  |   This is interpreted strongly as an OWL equivalence axiom, meaning the two terms are considered to be exactly the same. This can be added to any xref where this is true.   |   xrefs   |   N   |   MONDO:0100087 'familial Alzheimer disease' database_cross_reference: GARD:0000632 (refers to the disease in GARD) {source=MONDO:equivalentTo}
 MONDO:includedEntryInOMIM | This indicates a term is an 'included' entry in OMIM, for example https://omim.org/entry/233910 | xrefs | N | MONDO:0100167 'dystonia, dopa-responsive, with or without hyperphenylalaninemia, autosomal recessive'
+MONDO:omim_formerly | Used on OMIM synonyms whose labels end with ", formerly", denoting a status similar to `owl:deprecated`. | synonyms | N | 
```

Comment:
I am not sure this is the correct description, e.g. `status similar to `owl:deprecated``. An example could be an OMIM that contains ", formerly".

Reply:
- [x] @joeflack4 An example could be an OMIM that contains ", formerly".
  - Done!
- [ ] @twhetzel / curators: Confirm/review notes on curator description of "formerly".

Reply:
@joeflack4 the definition of `omim_formerly` is something you can check on during a Mondo Tech call since curators are at the meeting

Reply:
Just noticed this, I think the casing should be `MONDO:omimFormerly`.

Reply:
suggested updated definition: 
Used on OMIM synonyms that have been deprecated in OMIM; in OMIM, these synonyms end with ", formerly".

@joeflack4

Reply:
@twhetzel Done!
- [x] Updated w/ Sabrina's verbiage.
- [x] Changed to camel case.